packages/protocol/blockchain_scripts/airdrop/2022-01-24-backers/calculation.ts

```diff
@@ -26,55 +22,74 @@ interface RawCSV {
 
 interface PreparedRow {
   backerAddress: string
-  immediatelyAvailable: BigNum,
+  immediatelyAvailable: BigNum
   vesting: BigNum
 }
 
 function sanitizeData(data: RawCSV[]): PreparedRow[] {
-  const toBigInt = (s: string) => new BigNum(s.replace(/,/g, ""))
-    .multipliedBy(decimals.toString())
+  const toBigInt = (s: string) => new BigNum(s.replace(/,/g, "")).multipliedBy(decimals.toString())
 
-  return data.map(row => ({
-      backerAddress: row['Backer Address'],
-      immediatelyAvailable: toBigInt(row['Immeditely available']),
-      vesting: toBigInt(row['Unlocks over 12 months']),
+  return data.map((row) => ({
+    backerAddress: row["Backer Address"],
+    immediatelyAvailable: toBigInt(row["Immeditely available"]),
+    vesting: toBigInt(row["Unlocks over 12 months"]),
   }))
 }
 
 async function saveImmediateData(data: PreparedRow[], path: string) {
   console.log(`Saving ${data.length} rows for no-vesting data to ${path}`)
-  await fs.writeFile(path, JSON.stringify(data.map(row => ({
-    account: row.backerAddress,
-    reason: LIQUIDITY_PROVIDER_GRANT_REASON,
-    grant: {
-      amount: row.immediatelyAvailable.toFixed(),
-      cliffLength: "0",
-      vestingInterval: "0",
-      vestingLength: "0",
-    }
-  })), null, 2))
+  await fs.writeFile(
```

Comment:
no actual change in any of these, is that right? Just changes from our linter?

Reply:
Yeah, just changes from running `npm run lint:fix`. Let's move these comments over to the original PR?